wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -22,6 +25,12 @@ function chld_thm_cfg_parent_css() {
 endif;
 add_action( 'wp_enqueue_scripts', 'chld_thm_cfg_parent_css', 10 );
 
+function ajax_scripts_method() {
+    wp_enqueue_script( 'student-ajax', get_stylesheet_directory_uri() . '/settings-student.js', array( 'jquery' ), DX_ASSETS_VERSION);
+    wp_localize_script( 'student-ajax', 'my_ajax_object', array( 'ajax_url' => admin_url( 'admin-ajax.php' ) ) );
+}
+add_action( 'admin_enqueue_scripts', 'ajax_scripts_method' );
+
 function add_my_filter( $content ) {
     $content = _e( apply_filters( 'custom_filter_change', "This is my filter" )) . $content;
```

Comment:
Okay I know this isn't part of the PR but `_e()`  actually directly displays the text https://developer.wordpress.org/reference/functions/_e/

What you need here is `__()`.